projects/core/src/lib/layers/symbology/raster-palette-symbology-editor/raster-palette-symbology-editor.component.ts

```diff
@@ -0,0 +1,186 @@
+import {Component, Input, ChangeDetectionStrategy, OnInit, ViewChild, Output, EventEmitter} from '@angular/core';
+import {RasterLayer} from '../../layer.model';
+import {MapService} from '../../../map/map.service';
+import {ProjectService} from '../../../project/project.service';
+import {Config} from '../../../config.service';
+import {BackendService} from '../../../backend/backend.service';
+import {PaletteColorizer} from '../../../colors/colorizer.model';
+import {ColorAttributeInput} from '../../../colors/color-attribute-input/color-attribute-input.component';
+import {UserService} from '../../../users/user.service';
+import {Color} from '../../../colors/color';
+import {ColorMapSelectorComponent} from '../../../colors/color-map-selector/color-map-selector.component';
+import {LayoutService} from '../../../layout.service';
+import {ColorTableEditorComponent} from '../../../colors/color-table-editor/color-table-editor.component';
+import {ColorBreakpoint} from '../../../colors/color-breakpoint.model';
+
+/**
+ * An editor for generating raster symbologies.
```

Comment:
Somehow, i can't set any colors in the table. It always uses the  no-data color. Tried the "green" layer, removed all entries and set 0 to black, 255 to white (1, and very large number also do not work)

Reply:
I fixed a bug that is related to this: [df53747](https://github.com/geo-engine/geoengine-ui/pull/647/commits/df537471142682c383fad3a039ae98fef0770f0e)

But I cannot reproduce this problem.